pkg/generators/models/testdata/cases/embedded_type/expected/model_sub.go

```diff
@@ -5,12 +5,25 @@
 //     Version: 0.1.0
 package generatortest
 
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+)
+
 // Sub is an object.
 type Sub struct {
 	// Foo:
 	Foo string `json:"foo,omitempty"`
 }
 
+// Validate implements basic validation for this model
+func (m Sub) Validate() error {
+	return validation.Errors{
+		"foo": validation.Validate(
+			m.Foo,
+		),
+	}.Filter()
+}
```

Comment:
This doesn't seem to do anythin, maybe we can skip it ?

Reply:
This does recursive validation for types that support it. Basically if the type of m.Foo implements Validatable then it gets validated. With this PR, all of our models are Validatable

Reply:
but Foo is just a string here, why not just return nil

Reply:
I could do a runtime check if it supports Validatable and only then call it, but that's exactly the same as just calling validate on everything

Reply:
it just feels like a validation rule for "foo", even an empty one just shouldn't be generated here

Reply:
The alternative would be check if the go type starts with a capital letter at generator runtime and then add it (to have the recursive validation) 
In other cases (string, float etc) we could indeed leave it out

Reply:
so, should be fixed now